apps/web/src/services/asset-inventory/collector/modules/CollectorContents.vue

```diff
@@ -52,66 +52,60 @@
 </template>
 
 <script setup lang="ts">
-import { computed, reactive, watch } from 'vue';
+import { computed, reactive } from 'vue';
 
 import {
     PToolbox, PButton, PCard,
 } from '@spaceone/design-system';
 
 import type { QueryTag } from '@cloudforet/core-lib/component-util/query-search/type';
-import type { ApiQueryHelper } from '@cloudforet/core-lib/space-connector/helper';
 
 import { SpaceRouter } from '@/router';
 import { store } from '@/store';
 
-import type { ReferenceItem } from '@/store/modules/reference/type';
-
-import { FILE_NAME_PREFIX } from '@/lib/excel-export';
+import type { ProviderReferenceMap } from '@/store/modules/reference/provider/type';
 
 import CollectorItemInfo from '@/services/asset-inventory/collector/modules/CollectorItemInfo.vue';
 import CollectorListNoData from '@/services/asset-inventory/collector/modules/CollectorListNoData.vue';
-import { CollectorItemInfoType } from '@/services/asset-inventory/collector/type';
+import { COLLECTOR_ITEM_INFO_TYPE } from '@/services/asset-inventory/collector/type';
 import ProviderList from '@/services/asset-inventory/components/ProviderList.vue';
 import { ASSET_INVENTORY_ROUTE } from '@/services/asset-inventory/route-config';
 import { useCollectorPageStore } from '@/services/asset-inventory/store/collector-page-store';
 
 interface Props {
-    queryHelper?: ApiQueryHelper
     totalCount?: number
     pageLimit?: number
 }
 
 const props = withDefaults(defineProps<Props>(), {
-    queryHelper: undefined,
     totalCount: 0,
     pageLimit: 15,
 });
 
 const cloudCollectorPageStore = useCollectorPageStore();
 const cloudCollectorPageState = cloudCollectorPageStore.$state;
 
+const storeState = reactive({
+    providers: computed<ProviderReferenceMap>(() => store.getters['reference/providerItems']),
+});
+
 const state = reactive({
-    providerList: [] as ReferenceItem[],
+    providerList: computed(() => ([{ key: 'all', name: 'All Providers' }, ...Object.values(storeState.providers)])),
```

Comment:
question: Does 'All providers' label not need any translation? 😯

Reply:
I checked the quest channel regarding this matter, but I haven't received a response yet. I will send notification again to confirm!

Reply:
I checked, and the label doesn't need to be translated!